src/script/lua_api/l_craft.cpp

```diff
@@ -347,10 +347,13 @@ int ModApiCraft::l_clear_craft(lua_State *L)
 	} else {
 		throw LuaError("Unknown crafting definition type: \"" + type + "\"");
 	}
+
 	if (!craftdef->clearCraftRecipesByInput(method, width, recipe, getServer(L)))
-		throw LuaError("No crafting specified for input");
+		warningstream << "No craft recipe matches input" << std::endl;
```

Comment:
missing `{}`

Reply:
Ugh ok this is why the correct form didn't work :)